src/core/main-page/main-page.component.tsx

```diff
@@ -5,7 +5,7 @@ const MainPage: FunctionComponent<Props> = ({className}) => {
   return (
     <div className={className}>
       <header className="title-container">
-        <h1>Game room</h1>
+        <h2>Game room</h2>
```

Comment:
me parece que el titulo deberia ser mas pequeño
```suggestion
        <h3>Game room</h3>
```